internal/daysteps/daysteps.go

```diff
@@ -1,23 +1,53 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength = 0.65
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	slice := strings.Split(data, ",")
+	if len(slice) != 2 {
+		return 0, 0, errors.New("ошибка")
```

Comment:
Лучше давать полный текст ошибки, чтобы было понятно суть проблемы
```suggestion
		return 0, 0, fmt.Errorf("the number of parameters must be 2")
```